terracumber_config/tf_files/SUSEManager-4.2-SLE-update.tf

```diff
@@ -0,0 +1,224 @@
+// Mandatory variables for terracumber
+variable "URL_PREFIX" {
+  type = string
+  default = "https://ci.suse.de/view/Manager/view/Manager-4.2/job/manager-4.2-qa-sle-update"
```

Comment:
Shouldn't this be `qe` instead of `qa` ? Same in the other .tf file.

Reply:
wow, you're right